apps/oxlint/src/lint.rs

```diff
@@ -107,9 +84,64 @@ impl Runner for LintRunner {
         let mut oxlintrc = config_search_result.unwrap();
         let oxlint_wd = oxlintrc.path.parent().unwrap_or(&self.cwd).to_path_buf();
 
-        let paths = Walk::new(&oxlint_wd, &paths, &ignore_options, &oxlintrc.ignore_patterns)
-            .with_extensions(Extensions(extensions))
-            .paths();
+        let mut override_builder = OverrideBuilder::new(&self.cwd);
```

Comment:
now we are building `OverrideBuilder` even when `ignore_options.no_ignore` is active.
Can you build it as an Option?